src/Operation/Watch.php

```diff
@@ -391,15 +391,7 @@ private function getInitialResumeToken()
             return $this->postBatchResumeToken;
         }
 
-        if (isset($this->changeStreamOptions['startAfter'])) {
-            return $this->changeStreamOptions['startAfter'];
-        }
-
-        if (isset($this->changeStreamOptions['resumeAfter'])) {
-            return $this->changeStreamOptions['resumeAfter'];
-        }
-
-        return null;
+        return $this->changeStreamOptions['startAfter'] ?? $this->changeStreamOptions['resumeAfter'] ?? null;
```

Comment:
Change created by `IfIssetToCoalescingRector`, can be deactivated if desired.

Reply:
We try not to use ternaries like this too much, because they make it harder to identify code paths not covered by the tests.

Reply:
Done